tests/test_encapsulation.rb

```diff
@@ -56,11 +56,15 @@ def test_encapsulation_dot1_mapping
                  'Error: dot1q is not matching')
     dot1q = '100-110'
     vni = '5000-5010'
-    encap.send(:set_dot1q_map=, true, dot1q, vni)
+    encap.dot1q_map = [dot1q, vni]
     assert_equal(dot1q, encap.dot1q_map[0].gsub(/\s+/, ''),
```

Comment:
This test tells me that the getter is not removing whitespace. It should be normalizing the state. Please fix the getter and change these tests so that they just use the raw getter result.